packages/ckeditor5-bookmark/src/bookmarkediting.ts

```diff
@@ -112,3 +127,25 @@ export default class BookmarkEditing extends Plugin {
 		} );
 	}
 }
+
+/**
+ * A helper function to match an `anchor` element which must contain `id` attribute but without `href` attribute,
+ * also element must be empty when matcher is execute in data pipeline so it can be upcasted to a `bookmark`.
+ */
+function upcastMatcher( element: ViewElement, dataPipeline: boolean = true ) {
```

Comment:
Why do we need `dataPipeline` argument, maybe it could goud idea do describe it in comment as @param?

Reply:
Addressed in 3f7a20f.